src/views/chart.js

```diff
@@ -2,7 +2,40 @@
 import React from "react";
 import { jsx } from "@emotion/core";
 
+const initialColaborators = JSON.parse(
+  localStorage.getItem("colaborators") ||
+    JSON.stringify([
+      {
```

Comment:
Lets set a really good model to test the drawing.

Reply:
Thanks for the advice, we will do it in the next pull request